content/post/articles/Become-A-Software-Developer.md

```diff
@@ -0,0 +1,98 @@
+
+The emphasis is much more on the theoretical aspect of the courses which sometimes displeases more than one.
+
+Do not forget one thing, the computer science degree is only a piece of paper, when you hire a developer, you hire him not for his beautiful eyes, but more for his skills, basically for what he is able to do.
+
+# Choose your BootCamp :
+
+{{<figure src="/Become-A-Software-Developer/Localhost academy.png"  alt="Image de la localHost Academy Douala">}}
+localhost academy of Douala 
+
+Coding bootcamps are intensive software development programs. They first appeared in 2011 (Wikipedia) .They are intensive, accelerated learning programs that teach beginners digital skills like Full-Stack Web Development, Data Science, Digital Marketing, UX / UI Design and more.Through project-based learning, coding bootcamps get students job-ready for a career in tech in generally about 12 weeks.At the end of the BootCamp, the BootCampers leave with a portfolio, an online presence, interview skills and more.
+
+##### #Pros:
+
+Provides skills in demand by businesses.
+
+Allows you to be hired quickly (it depends on your level of practice)
+
+BootCamps are much more affordable than universities.
+
+The duration of the training is much shorter.
+
+##### #Cons:
+
+Being trained for just 12 weeks in Software Development can be difficult due to the short duration of training, and for this reason it is not easy for everyone to adapt to it.
+
+Usually in bootcamps 90% of learning is done hands-on, but some people feel that the main emphasis should be on concept theory as they find it much more interesting and important than practice.
+
+# Be a full Sel-Taught developer:
+
+{{<figure src="/Become-A-Software-Developer/self-taught-developer.jpg"  alt="self-taught-developer">}}
+
+A self-taught developer doesn't wait for someone to take initiative on their behalf and prepare to excel in programming skills of any kind. When you learn to code, you learn at your own pace using free or paid online / offline resources.
+
+##### #Pros:
+
+Full developer Self-Taught is much more free we all agree on this, he has no teachers on his back, or an instructor everything happens between him, his computer and obviously the internet (it is better)
+His study program, his projects, his ideas are all from A to Z managed by him, he is taken into consideration much more when he has assimilated the different concepts and principles of software development.
+
+##### #Cons:
+
+The self-taught developer is taken much less seriously because the recruiter does not have confidence in his background, for this reason a self-taught developer must have a concrete portfollio, be positive, have self-confidence, know how to express his programming skills , his communication skills, he must be persevering.
```

Comment:
```suggestion
The self-taught developer is taken much less seriously because the recruiter does not have confidence in his background, for this reason a self-taught developer must have a concrete portfolio, be positive, have self-confidence, know how to express his programming skills, his communication skills, he must be persevering.
```